frontend/src/modules/S3_Datasets/views/DatasetCreateForm.js

```diff
@@ -494,45 +502,103 @@ const DatasetCreateForm = (props) => {
                       <Card>
                         <CardHeader title="Governance" />
                         <CardContent>
-                          <TextField
-                            fullWidth
-                            error={Boolean(
-                              touched.SamlGroupName && errors.SamlGroupName
+                          <Autocomplete
+                            id="SamlAdminGroupName"
+                            disablePortal
+                            options={groupOptions.map((option) => option)}
+                            onChange={(event, value) => {
+                              if (value && value.value) {
+                                setFieldValue(
+                                  'SamlAdminGroupName',
+                                  value.value
+                                );
+                              } else {
+                                setFieldValue('SamlAdminGroupName', '');
+                              }
+                            }}
+                            inputValue={values.SamlAdminGroupName}
+                            renderInput={(params) => (
+                              <Box>
+                                {groupOptions.length > 0 ? (
+                                  <TextField
+                                    {...params}
+                                    fullWidth
+                                    error={Boolean(
+                                      touched.SamlAdminGroupName &&
+                                        errors.SamlAdminGroupName
+                                    )}
+                                    helperText={
+                                      touched.SamlAdminGroupName &&
+                                      errors.SamlAdminGroupName
+                                    }
+                                    label="Team"
+                                    variant="outlined"
+                                  />
+                                ) : (
+                                  <TextField
+                                    error={Boolean(
+                                      touched.SamlAdminGroupName &&
+                                        errors.SamlAdminGroupName
+                                    )}
+                                    helperText={
+                                      touched.SamlAdminGroupName &&
+                                      errors.SamlAdminGroupName
+                                    }
+                                    fullWidth
+                                    disabled
+                                    label="Team"
+                                    value="No teams found for this environment"
+                                    variant="outlined"
+                                  />
+                                )}
+                              </Box>
```

Comment:
nit: This TextField component (and all other ones for when groupOptions is empty) has a different sizing then the other TextFields on the form and the other TextField when groupOptions are present

Reply:
I noticed, not very sure why